frontend/src/components/pages/WaitlistExperience/PersonalInfo/index.tsx

```diff
@@ -1,19 +1,288 @@
 import React from "react";
-import { Checkbox } from "@chakra-ui/react";
+import { SmallAddIcon } from "@chakra-ui/icons";
+import { Box, Button, Divider, Text, VStack } from "@chakra-ui/react";
+import { CampResponse } from "../../../../types/CampsTypes";
+import { CreateWaitlistedCamperDTO } from "../../../../types/CamperTypes";
+import {
+  checkAge,
+  checkEmail,
+  checkFirstName,
+  checkLastName,
+  checkPhoneNumber,
+  usePersonalInfoDispatcher,
+} from "./personalInfoReducer";
+import PersonalInfoCard, {
+  CamperCardInputTypes,
+} from "../../../common/personalInfoRegistration/PersonalInfoCard";
+import CamperCardDeletionButton from "../../../common/personalInfoRegistration/CamperCardDeletionButton";
+import { PersonalInfoActions } from "../../../../types/PersonalInfoTypes";
 
 type WaitlistPersonalInfoPageProps = {
-  isChecked: boolean;
-  toggleChecked: () => void;
+  nextBtnRef: React.RefObject<HTMLButtonElement>;
+  campers: CreateWaitlistedCamperDTO[];
+  camp: CampResponse;
+  setCampers: React.Dispatch<React.SetStateAction<CreateWaitlistedCamperDTO[]>>;
 };
 
 const WaitlistPersonalInfoPage = ({
-  isChecked,
-  toggleChecked,
+  nextBtnRef,
+  campers,
+  camp,
+  setCampers,
 }: WaitlistPersonalInfoPageProps): React.ReactElement => {
+  const dispatchPersonalInfoAction = usePersonalInfoDispatcher(setCampers);
   return (
-    <Checkbox isChecked={isChecked} onChange={toggleChecked}>
-      Waitlist confirmation step
-    </Checkbox>
+    <Box pb={14} px="10%">
```

Comment:
![image](https://user-images.githubusercontent.com/30396273/214670111-829083f2-4220-4cd9-8900-574a178e05a2.png)
Can you make the space between the title and the bottom of the navbar smaller? Try to make it the same as the bottom of the navbar and the title for the registration flow. You'll probably need some responsive code, since I think it looks fine for mobile, and just a bit too low for tablet.